extensions/test/CognitoAuthentication.IntegrationTests/_coreclr/BaseAuthenticationTestClass.cs

```diff
@@ -0,0 +1,47 @@
+﻿/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
please update all copyright notices to 2018.

also, doesn't this file require update to [CognitoAuthentication.IntegrationTests.CoreCLR.csproj](https://github.com/aws/aws-sdk-net/blob/master/extensions/test/CognitoAuthentication.IntegrationTests/CognitoAuthentication.IntegrationTests.CoreCLR.csproj) ?   As it is the .csproj is picking up both copies of the BaseAuthenticationTestClass.cs, no?

Reply:
We don't have to, as the source _coreclr\BaseAuthenticationTestClass.cs will be included to project CognitoAuthentication.IntegrationTests.CoreCLR.csproj automatically. CognitoAuthentication.IntegrationTests.CoreCLR.csproj will include all files under the project folder and its sub folders to the project